Makefile

```diff
@@ -181,23 +181,38 @@ tests/%.uplc.run: tests/%.uplc
 	$(KPLUTUS) run $< > $<.out
 	$(CHECK) $<.out $<.expected
 
+# Compare K output against expected output generated from the uplc tool
+# Do this by extracting the contents of the K cell.
+tests/%.uplc.runConformance: tests/%.uplc
+	$(KPLUTUS) run $< |\
+	tr '\n' ' ' | sed "s/.*<k>\(.*\)~> \. .*/\1/g" > $<.out
+	$(CHECK) $<.out $<.expected
```

Comment:
I think `tr` has some compatibility issues with MacOS.

Either way, I would lean towards adding a `--result-only` options to `kplc run ...`, instead of doing this in the Makefile. It may be useful option in other circumstances too. So basically just make a new flag `--result-only`, and if this is passed then process the output of `krun ...` in this same way.

Eventually, we can likely make this much faster by just having K only unparse the `<k>` cell.

Reply:
I was looking at how to do this using sed and it looked like it was using a feature specific to gnu sed. mac has a freebsd variant of sed. This tr command seems pretty basic so I left it in my new commit... Either way, we should get a mac user to test this..